airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -83,6 +88,9 @@ def check_local_docker_configuration() -> None:
 
 
 def is_dagger_run_enabled_by_default() -> bool:
+    if CI_REQUIREMENTS_OPTION_NAME in sys.argv:
```

Comment:
💅 Note for future: We do need to invert this at some point where commands specifiy if they need dagger run or not.